packages/@lwc/babel-plugin-component/src/index.js

```diff
@@ -43,6 +44,9 @@ module.exports = function LwcClassTransform(api) {
 
                     // Validate the usage of LWC decorators.
                     validateImportedLwcDecoratorUsage(engineImportSpecifiers);
+
+                    // Add ?scoped=true to *.scoped.css imports
+                    scopeCssImports(api, path);
```

Comment:
This compiler change is not behind the flag, but I think the impact is very small so it shouldn't matter. Any bundler capable of dealing with `?scoped=true` imports from CSS should be able to deal with them from JS as well.

Technically, though, this does allow people to use scoped styles with the `template.stylesheets = [...]` hack. But people are also able to do this with arbitrary functions anyway. 😛  #3122